### DISCUSSION TITLE
如何以web形式免费发布/公开笔记？

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#4] [Q&A常见问题]
尽管obsidian官方有publish的功能，不过[费用](https://obsidian.md/pricing)稍有些昂贵，你可以在这里分享 **免费发布/公开笔记** 的方法！欢迎留下你的解决方案

**Comment by philubuntutu** [comment]
为这个问题找到这个论坛，原来还没有官方之外的解决方法吗？

就像Matters这种支持markdown语言的平台，好像也没法用图片链接直接发布……

**Comment by dreamkidd** [comment]
如果不想用 官方的 publish 的话 可以考虑 pandoc 转 html 然后发布在自己的 html ?

**Comment by revervd1** [comment]
可以尝试去知乎一些网站看看

**Comment by gledos** [comment]
我使用的是 MkDocs，模板可以查看这个仓库 [Jackiexiao/foam-mkdocs-template](https://github.com/Jackiexiao/foam-mkdocs-template)，不过细节大概就得研究 mkdocs 和 mkdocs-material 了。

**Comment by lefinite** [comment]
其实社区有很多插件能满足，比如这个：https://github.com/oleeskild/Obsidian-Digital-Garden
